packages/storybook/src/nimble/select/select.mdx

```diff
@@ -42,6 +42,15 @@ in addition to selecting from the list.
 <Description of={listOptionGroupStories} />
 <Controls of={listOptionGroupStories.listOptionGroup} />
 
+### Angular
```

Comment:
Looks like the example app follows this pattern. Should we update it to obey the new guidance? Or update the guidance to explain whatever subtlety makes it so the bug doesn't occur in that case?

https://github.com/ni/nimble/blob/3b0f6f6a149e769fb040b5b1331154d9709327fb/packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.html#L199